Kickstarter-iOS/Views/Controllers/SettingsViewController.swift

```diff
@@ -313,7 +313,7 @@ internal final class SettingsViewController: UIViewController {
 
     _ = self.wereAllEarsTitleLabel
       |> settingsTitleLabelStyle
-      |> UILabel.lens.text %~ { _ in Strings.profile_settings_rating_title() }
+      |> UILabel.lens.text %~ { _ in Strings.We_re_all_ears() }
```

Comment:
Can we chat with Sara about an alternate expression here or maybe lose it altogether? I think the period instead of an exclamation makes this title feel awkwardly serious.

<img width="845" alt="screen shot 2017-04-03 at 11 34 53 am" src="https://cloud.githubusercontent.com/assets/988998/24617436/de857456-1861-11e7-8d42-2557d214eefb.png">